src/SFA.DAS.EmployerIncentives.Data/IncentiveApplication/IncentiveApplicationQueryRepository.cs

```diff
@@ -63,21 +68,31 @@ public Task<List<IncentiveApplicationDto>> GetList(Expression<Func<IncentiveAppl
 
         private static IncentiveApplicationApprenticeshipDto MapToApprenticeshipDto(Models.IncentiveApplicationApprenticeship apprenticeship)
         {
+            var phase = apprenticeship.Phase;
+            if (phase == Phase.NotSet)
+            {
+                phase = Phase.Phase2;
```

Comment:
Should this be Phase3 now and merged into the feature branch?

Reply:
That's been refactored now, as the payment profiles service was removed as part of EI-1205. It now gets the payment profiles from the incentive.